tests/hamcrest_unit_test/core/isinstanceof_test.py

```diff
@@ -23,6 +23,8 @@ class IsInstanceOfTest(MatcherTest):
     def testEvaluatesToTrueIfArgumentIsInstanceOfASpecificClass(self):
         self.assert_matches('same class', instance_of(int), 1)
 
+        self.assert_matches('same class', instance_of((int, long)), 1)
```

Comment:
I'd like to see something that tests with the second argument; maybe `instance_of((int, datetime), datetime.now())`